features/org.csstudio.dls.product.feature/rootfiles/css.sh

```diff
@@ -17,6 +17,12 @@ CS-Studio running, then the workspace argument is ignored.
     "
 }
 
+function escape() {
+    # CSS cannot accept : or . (other than in the filename) in a command-line argument.
```

Comment:
This comment probably needs to be edited to be more precise, as I didn't fully understand when I wrote it.